Source/ACE/Entity/Landblock.cs

```diff
@@ -329,6 +324,11 @@ public void HandleMovementEvent(WorldObject sender, GeneralMotion motion)
             BroadcastEventArgs args = BroadcastEventArgs.CreateMovementEvent(sender, motion);
             Broadcast(args, true, Quadrant.All);
         }
+        public void HandleMovementEvent(WorldObject sender, ServerControlMotion motion)

```

Comment:
I will refactor it to work that way.   Just lack of experience on my part.   Thanks for the feedback and direction.